.github/policies/IssueManagement.ResolveStale.yml

```diff
@@ -103,5 +120,14 @@ configuration:
       then:
       - removeLabel:
           label: KeepOpen
+
+    - description: If new issue comment then remove needs repro in case it includes repro steps
```

Comment:
Maybe it should add "needs-attention" when removing "needs-repro", so it's easier to find issues that may have repro steps added.